### ISSUE TITLE
Rust hello world

### ISSUE THREAD

**Issue by user-name-is-taken** [user-name-is-taken/jsdr#1]
Get the rust to wasm compilers workin https://developer.mozilla.org/en-US/docs/WebAssembly/Rust_to_wasm

**Comment by potto216** [comment]
Yeah I followed this tutorial for rust in WASM which worked: https://rustwasm.github.io/book/game-of-life/introduction.html . 
I can commit an example which we can build off for the SDR code

**Comment by potto216** [comment]
Before committing the wasm example what should the directory structure be? Something like:
* jsdr #the main app which combines SDR hardware->collection (webusb)->DSPprocessing(wasm/rust)-> ui (Javascript) 
* ex # examples of the parts that make up the main app
    * collection
      * rtl-sdr *configure rtl-sdr and simple connectivity test*
      * hackrf *configure hackrf and simple connectivity test*
    * dsp
      *  iq-fir-filter *iq filter filter structure*
    * ui
      *  plot-iq *plot data*

**Comment by user-name-is-taken** [comment]
Whatever's probably good but probably kick off the compiling with a package.json from npm
